pkg/rhRegion/rhRegion.go

```diff
@@ -0,0 +1,50 @@
+package rhRegion
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	sdk "github.com/openshift-online/ocm-sdk-go"
+	"os"
+	"strings"
+)
+
+type Region struct {
+	URL string
+	AWS []string
+	GCP []string
+}
+
+func GetRhRegions(ctx context.Context, token string) (map[string]Region, error) {
+	var regions map[string]Region
+	globalConn, err := sdk.NewConnectionBuilder().Tokens(token).RhRegion("integration").BuildContext(ctx)
```

Comment:
couple things:
1. ocm-shards.json is public, its doesnt actually need a token, you just need a url
2. forget about RhRegion on the connection builder, it should probably be removed, it doesnt make sense there

So see if you can make this function signature work `func GetRhRegions(ctx context.Context, URL String) (map[string]Region, error)`. And then later we can try to refactor `GetRhRegions` and `GetRhRegion` down into the ocm-sdk itself.

Reply:
Changed sdk call to simple http get method